### PR TITLE
percona-server 5.7.10-1rc1 (devel)

### PR DESCRIPTION
**Pull request by teslamint**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Looks good to me. :+1: once green.

**Comment by teslamint** [comment]
I think it need some cleanup(about TokuDB option, especially) after #47756 merged.

File: Library/Formula/percona-server.rb [hunk 1]
```diff
@@ -5,6 +5,19 @@ class PerconaServer < Formula
...
+    url "https://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.10-1rc1/source/tarball/percona-server-5.7.10-1rc1.tar.gz"
+    version "5.7.10-1rc1"
+    sha256 "68de79af4a761522f436b89eeb5bbb43a56dccc2bc352d38c79b6e6be0c4106c"
+
+    depends_on "zlib" => :build
+
+    resource "boost" do
+      url "https://downloads.sourceforge.net/project/boost/boost/1.59.0/boost_1_59_0.tar.bz2"
+      sha256 "727a932322d94287b62abb1bd2d41723eec4356a7728909e38adb65ca25241ca"
+    end
```
Comment (nijikon) [inline Library/Formula/percona-server.rb]: Why this can't be ```depends_on "boost"```?

**Comment by nijikon** [comment]
:+1:

**Comment by teslamint** [comment]
CI El Captian tests timeout: https://travis-ci.org/Homebrew/homebrew/jobs/103077054

File: Library/Formula/percona-server.rb [hunk 2]
```diff
@@ -5,6 +5,19 @@ class PerconaServer < Formula
   version "5.6.25-73.1"
   sha256 "5a0d88465e4bb081e621b06bc943fafadb4c67a2cca50839b44fcd94ae793b50"
 
+  devel do
+    url "https://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.10-1rc1/source/tarball/percona-server-5.7.10-1rc1.tar.gz"
+    version "5.7.10-1rc1"
+    sha256 "68de79af4a761522f436b89eeb5bbb43a56dccc2bc352d38c79b6e6be0c4106c"
+
+    depends_on "homebrew/dupes/zlib" => :build
```
Comment (DomT4) [inline Library/Formula/percona-server.rb]: This is slightly weird. Why does it require `zlib` from dupes to build?

**Comment by DomT4** [comment]
You'll need to rebase this against the master now I'm afraid, we merged in a stable update earlier which has caused a conflict here.

File: Library/Formula/percona-server.rb (hunk 2 above)
Comment (teslamint) [inline Library/Formula/percona-server.rb]: I'll try build again without that.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/09236b3d8839f3a973b3479bbd38b9f43161ee08. Thank you for another Homebrew contribution @teslamint! :smiley_cat:

**Pull request closed without merging**